### PR TITLE
init istio-monitoring chart

### PR DESCRIPTION
**Pull request by zirain**
- add grafana and prometheus as subchart
- import oss dashboards

### CODE REVIEW

File: charts/istio-monitoring/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
Comment (psbrar99) [inline charts/istio-monitoring/Chart.yaml:24]: @zirain : is this the version we want to start for the istio-monitoring chart?

**Review by psbrar99 — COMMENTED** [review]
overall looks good. Just wanted to confirm the `appVersion: "1.16.0"`

File: charts/istio-monitoring/Chart.yaml (hunk 1 above)
Comment (zirain) [reply charts/istio-monitoring/Chart.yaml:24]: it's a nit.

**Review by psbrar99 — APPROVED** [review]

**Comment by psbrar99** [comment]
@zirain : can you add README.md for these charts. I think [this](https://docs.google.com/document/d/1C2AiMd7AY1F9zujinjtDPT0RNKut1t_ZMlALIh9nSUM/edit#heading=h.y4d9d6acbn7r) is good enough.

**Comment by zirain** [comment]
> @zirain : can you add README.md for these charts. I think [this](https://docs.google.com/document/d/1C2AiMd7AY1F9zujinjtDPT0RNKut1t_ZMlALIh9nSUM/edit#heading=h.y4d9d6acbn7r) is good enough.

will do

**Review by psbrar99 — APPROVED** [review]
LGTM!
Thanks.
once we have everyone onboard with [this](https://github.com/tetrateio/tid-internal/issues/87), we can publish the charts in the repo.

**Pull request merged**